discuss/customizations/mail_activity_history.rst

```diff
@@ -0,0 +1,45 @@
+=====================
+History of Activities
+=====================
+
+Save done and cancelled activities in history table for later analysis.
+
+Configuration
+=============
+
+For these features to work, module :code:`mail_activity_history` must be
+installed.
+
+.. note::
+    In multi-company environment, module
+    :code:`mail_activity_history_multicompany` should be installed, so
+    users could access only global or user's company history.
+
+Log Cancelled Activity
+======================
+
+Optionally, it might be configured to log cancelled activities in the
+chatter, just like done activities. For this feature to activate,
```

Comment:
`For this feature to activate` sounds weird. Like it will activate itself. I would rephrase like `To activate this feature, do this and that`.